packages/default/scss/pager/_layout.scss

```diff
@@ -199,9 +201,6 @@
         }
     }
 
-
-
-
     // Responsive
     .k-pager-lg,
     .k-pager-md,
```

Comment:
I believe these styles are now relevant only to "-sm" page, we should remove the "-md" selector, as it gives margins to the "k-pager-numbers-wrap", etc.